### PR TITLE
Widget Visibility: add scaffolding for upcoming editor features.

### PR DESCRIPTION
**Pull request by jeherve**
#### Changes proposed in this Pull Request:
﻿
This does not much on its own, but is the first step to allow us to ship #20731.

This will also allow us to get Jetpack and WordPress.com Simple in sync, for easier merges and changes in the future.

#### Jetpack product discussion

* pbAok1-2dI-p2

#### Does this pull request change what data or activity we track or use?

* No

#### Testing instructions:

* Is CI happy?

### CODE REVIEW

**Comment by jeherve** [comment]
r231054-wpcom

File: projects/plugins/jetpack/modules/widget-visibility/editor/README.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Widget Visibility editor controls
+
+[Widget Visibility](http://jetpack.com/support/widget-visibility) was first built to interact with widgets, in a WordPress world where the block editor didn't exist.
+In WordPress 5.8, it became possible to [use a block-based editor to manage widgets on your WordPress site](https://wordpress.org/news/2021/08/widgets-in-wordpress-5-8-and-beyond/).
+
+This means that we can now develop a new interface to manage widget visibility settings from the block editor. This directory allows us to build that interface.
+
+## Build (& Watch)
+
+This feature will be built via `pnpm run build-widget-visibility`, which is invoked in `pnpm build`.
```
Comment (anomiex) [inline projects/plugins/jetpack/modules/widget-visibility/editor/README.md:10]: Should this be encouraging `pnpm build`?
```suggestion
This feature will be built via `pnpm run build-widget-visibility`, which is invoked in `jetpack build plugins/jetpack`.
```
But that can be fixed in a followup.

**Review by anomiex — APPROVED** [review]
CI looks happy. Let's go ahead with this to get things back in sync.

File: projects/plugins/jetpack/modules/widget-visibility/editor/README.md (hunk 1 above)
Comment (jeherve) [reply projects/plugins/jetpack/modules/widget-visibility/editor/README.md:10]: You're right. #20935

**Pull request merged**